RecoLocalMuon/GEMRecHit/plugins/GEMRecHitProducer.cc

```diff
@@ -31,6 +31,9 @@ GEMRecHitProducer::GEMRecHitProducer(const ParameterSet& config)
 
   // Turns off GE2/1 demonstrator reconstruction in Run3
   ge21Off_ = config.getParameter<bool>("ge21Off");
+  if (ge21Off_)
```

Comment:
@yeckang I am not sure to understand the logic since I find it misleading. If ge21Off_ = True, turn OFF is ENABLED and the GE2/1 demonstrator collections are produced?